BasketTest.md

```diff
@@ -1,5 +1,16 @@
 # Shopping Basket Discount Task
 
+## What is the Task?
+The coding exercise is the second stage of our technical interview process. We want it to be as real world as possible so research and use the resources necessary. 
+
+Feel free to take as long as you feel you need to on this task, we understand that not everyone has a huge amount of time while others feel they want to spend longer to demonstrate everything they can. 
+
+Note your decisions, constraints and thoughts in a readme to help us understand your process.
+
+## Why do we ask for the Task?
+
+The coding exercise allows us to view your work in the most realistic way we can giving a chance for you to show us your problem solving skills. There are many approaches to the task and being able to talk over the approach, options and considerations in the final stage of our interview.
```

Comment:
```suggestion
The coding exercise allows us to view your work in the most realistic way we can, giving you a chance to show us your problem solving skills. There are many approaches to the task. Being able to talk over the approach, options and considerations in the final stage of our interview helps us understand your thought process.
```

Reply:
Do we suggest that the candidate will get through to the final stage here, regardless of standard of basket test?